### ISSUE TITLE
Add language code `la` Latin

### ISSUE THREAD

**Issue by mksinicus** [tateisu/SubwayTooter#253]
Hi! Thanks a lot for the great app.

Not sure about other servers, but <mstdn.social> has `la` language code available for toots. I found it missing from SubwayTooter's (v 5.537) language drop-down list (should precede `lb` Luxembourgish). Can it be added?

Would highly appreciate it!

**Comment by tateisu** [comment]
The list of language codes currently displayed by ST is obtained from inside the device. This may differ from the language list that Ruby has on the server side.
Mastodon already has a PR to allow getting the list of language codes from the API. Only after it has been released can ST be modified to refer to it.

**Comment by tateisu** [comment]
https://github.com/mastodon/mastodon/pull/24443

**Comment by mksinicus** [comment]
Understood! Thanks for clarifying. :heart: 

So I guess I shall leave this issue open as a remainder? ;)

**Comment by ConfuSomu** [comment]
Perhaps instead of a querying the Mastodon instance for available languages, the user could additionally be able to enter a language code to use for the post? This would be faster than scrolling through the long list of languages (maybe this could be a separate issue).

Nevermind, as this API endpoint has been merged into Mastodon's master branch and is available since Mastodon 4.2.0.

Thank you!
